pkg/helmcli/repository_client.go

```diff
@@ -0,0 +1,31 @@
+package helmcli
+
+import (
+	"context"
+
+	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
+
+	"helm.sh/helm/v3/pkg/cli"
+)
+
+type RepositoryClient interface {
+	NewAdder(flags.AddFlags) (Adder, error)
+}
+
+type Adder interface {
```

Comment:
Can we abstract this interface into a subpackage instead of defining this in helmcli package, since this is public?

Reply:
Will move it to helmcli/repository